### ISSUE TITLE
Lssap parser fails to parse output from lssap command

### ISSUE THREAD

**Issue by bfahr** [RedHatInsights/insights-core#2942]
Parser `Lssap` throws a `ParseException` with what appears to be valid output from the `lssap` command.  Seems this should be a `SkipComponent` exception instead.

```
ParseException: Lssap: Unable to parse 10 line(s) of content: ([u'*** ERROR => CTrcOpen: fopen dev_lssap', u'', u'---------------------------------------------------', u'trc file: "dev_lssap", trc level: 1, release: "721"', u'---------------------------------------------------', u' - lssap version 1.0 -', u'==========================================', u'  SID   Nr   Instance    SAPLOCALHOST                        Version                 DIR_EXECUTABLE', u'  D02|  50|       D50|     sapcbapp08|722, patch 201, changelist 1718183|          /usr/sap/D02/D50/exe', u'=========================================='])
```

**Comment by flowersw** [comment]
Just curious here...

would raising `SkipComponent` instead of `ParseException` be a stop-gap, to allow the parser to continue, until a review of why it doesn't seem to be parsing seemingly valid input is completed? And if it's that simple of a change, happy to contribute that.

**Comment by bfahr** [comment]
This was fixed by #2969
